docs/source/release-notes.rst

```diff
@@ -3,17 +3,140 @@ FiftyOne Release Notes
+
+.. _release-notes-v1.1.0:
+
+FiftyOne 1.1.0
+--------------
+*Released December 3, 2024*
+
+What's New
+
+- Added a :ref:`Model Evaluation panel <app-model-evaluation-panel>` for
+  visually and interactively evaluating models in the FiftyOne App
+- Introduced :ref:`Query Performance <app-optimize-query-performance>` in the
+  App, which automatically nudges you to create the necessary indexes to
+  greatly optimize queries on large datasets
+- Added a :ref:`leaky splits method <brain-leaky-splits>` for automatically
+  detecting near-duplicate samples in different splits of your datasets
+- Added a :ref:`near duplicates method <brain-near-duplicates>` that scans
+  your datasets and detects potential duplicate samples
+
+App
+
+- Added zoom-to-crop and set-look-at for selected labels in the
+  :ref:`3D visualizer <app-3d-visualizer>`
+  `#4931 <https://github.com/voxel51/fiftyone/pull/4931>`_
+- Gracefully handle deleted + recreated datasets of the same name
+  `#5183 <https://github.com/voxel51/fiftyone/pull/5183>`_
+- Fixed a bug that prevented video playback from working for videos with
+  unknown frame rate
+  `#5155 <https://github.com/voxel51/fiftyone/pull/5155>`_
+
+SDK
+
+- Added :meth:`min() <fiftyone.core.collections.SampleCollection.min>` and
+  :meth:`max() <fiftyone.core.collections.SampleCollection.max>` and
+  aggregations
+  `#5029 <https://github.com/voxel51/fiftyone/pull/5029>`_
+- Improved support for creating summary fields and indexes
+  `#5091 <https://github.com/voxel51/fiftyone/pull/5091>`_
+- Added support for creating compound indexes when using the builtin
+  :class:`create_index <fiftyone.operators.builtin.CreateIndex>` operator that
+  optimize sidebar queries for group datasets
+  `#5174 <https://github.com/voxel51/fiftyone/pull/5174>`_
+- The builtin
+  :class:`clear_sample_field <fiftyone.operators.builtin.ClearSampleField>`
+  and
+  :class:`clear_frame_field <fiftyone.operators.builtin.ClearFrameField>`
+  operators now support clearing fields of views, in addition to full datasets
+  `#5122 <https://github.com/voxel51/fiftyone/pull/5122>`_
+- Fixed a bug that prevented users with `pydantic` installed from loading the
+  :ref:`quickstart-3d dataset <dataset-zoo-quickstart-3d>` from the zoo
+  `#4994 <https://github.com/voxel51/fiftyone/pull/4994>`_
+
+Brain
+
+- Added support for passing existing
+  :ref:`similarity indexes <brain-similarity>` to
+  :func:`compute_visualization() <fiftyone.brain.compute_visualization>`,
+  :func:`compute_uniqueness() <fiftyone.brain.compute_uniqueness>`, and
+  :func:`compute_representativeness() <fiftyone.brain.compute_representativeness>`
+  `#201 <https://github.com/voxel51/fiftyone-brain/pull/201>`_,
+  `#204 <https://github.com/voxel51/fiftyone-brain/pull/204>`_
+- Upgraded the :ref:`Pinecone integration <pinecone-integration>` to support
+  `pinecone-client>=3.2`
+  `#202 <https://github.com/voxel51/fiftyone-brain/pull/202>`_
+
+Plugins
```

Comment:
Added [skip_prompt](https://github.com/voxel51/fiftyone/blob/a6b5481a6f5627ac201fa30bbaa8126bda5cb2d3/fiftyone/operators/executor.py#L757) - that should be in here.

Reply:
Added thanks 👍